src/main/kotlin/de/bund/digitalservice/useid/widget/WidgetController.kt

```diff
@@ -76,40 +87,48 @@ class WidgetController(
     }
 
     @GetMapping("/$FALLBACK_PAGE")
-    fun getUniversalLinkFallbackPage(model: Model, @RequestParam tcTokenURL: String, @RequestParam(required = false, name = "hash") sessionHash: String?, @RequestHeader(name = HttpHeaders.USER_AGENT, required = false) userAgent: String?): ModelAndView {
+    fun getUniversalLinkFallbackPage(
+        model: Model,
+        @RequestParam tcTokenURL: String,
+        @RequestParam(required = false, name = "hash") sessionHash: String?,
+        @RequestAttribute tenant: Tenant,
+        @RequestHeader(name = HttpHeaders.USER_AGENT, required = false) userAgent: String?,
+    ): ModelAndView {
         publishMatomoEvent(
             widgetTracking.actions.loaded,
             widgetTracking.names.fallback,
             sessionHash,
             userAgent,
+            tenant.id,
         )
         /*
             Documentation about the link syntax can be found in Technical Guideline TR-03124-1 – eID-Client, Part 1:
             Specifications Version 1.4 8. October 2021, Chapter 2.2 Full eID-Client
             Note: Replaced the prefix eid:// with bundesident:// to make sure only the BundesIdent app is opened
          */
-        val url = "bundesident://127.0.0.1:24727/eID-Client?tcTokenURL=${URLEncoder.encode(tcTokenURL, UTF_8)}"
-
         val widgetViewFallbackConfig = mapOf(
-            setEiDClientURL(url),
+            "eidClientURL" to "bundesident://127.0.0.1:24727/eID-Client?tcTokenURL=${URLEncoder.encode(tcTokenURL, UTF_8)}",
```

Comment:
To be consistent with the rest of the code base, we'd have to move the base url out to a constant.